axis_backend/apps/kpis/views/kpi_viewset.py

```
@@ -0,0 +1,109 @@
+        """Get global KPIs (not client/contract specific)."""
+        kpis = self.service.get_global_kpis()
+        serializer = self.list_serializer_class(kpis, many=True)
+        return Response(serializer.data)
+
+    @extend_schema(
+        summary="Toggle KPI visibility",
+        tags=["KPIs"],
+        responses={200: KPIDetailSerializer}
+    )
+    @action(detail=True, methods=['post'])
+    def toggle_visibility(self, request, pk=None):
```

Ruff (0.14.5)

Unused method argument: `request`

(ARG002)